src/openzaak/components/documenten/tests/test_validation.py

```diff
@@ -14,6 +14,7 @@
 from .factories import EnkelvoudigInformatieObjectFactory
 
 
+@override_settings(ALLOWED_HOSTS=["testserver"])
```

Comment:
FYI - it can't hurt, but the django test runner inserts `testserver` into ALLOWED_HOSTS already :-)

Reply:
Yes, but I suppose it also keeps wildcards, i.e. all the urls in loose-fk fields are processed as locals
So this test doesn't work without `override_settings`

Reply:
Yep, correct. Because of the `dev` settings having the wildcard. The system check should've complained about that, no?

Reply:
Yes, but it's just a warning message :)